src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -119,6 +119,7 @@ namespace winrt::TerminalApp::implementation
         if (auto page{ weakThis.get() })
         {
             _UpdateCommandsForPalette();
+            CommandPalette().SetKeyBindings(_settings->GetKeybindings());
```

Comment:
This isn't actually "set the bindings on the command palette", this is "attach the combined [key binding list and key binding key-receiver action dispatcher] to the palette" which is _absolutely required_ for the palette to _dispatch_ key binding events.

Reply:
if we had a better separation of responsibilities here, we would be able to give command palette a key binding handler whose _only bound actions_ were next/prev tab

Reply:
(and then it could handle them itself). but right now our key binding command dispatch interface is an interface containing 100 functions it would need to implement **and** it is tied up with the actual deserializer code. heck.

Reply:
Carlos, looks like you've made a couple comments below about the same keybindings topic so I'll move all the discussion here:

Yeah I honestly didn't know how to trim down the given bindings to just nextTab and prevTab 😢
I'm mostly hoping that because the control needs to be kept open with a modifier key, and there's no search bar and the only interactable interface is a list view, the range of actions a user would _want_ to do is limited.

Like, if you've hit ctrl+tab to open the tab switcher, and for some odd reason you shift your fingers to invoke find which could be something like ctrl+shift+f, it'll show the find dialog and leave the tab switcher open. But IMO that's pretty inconvenient considering the whole time you'd have to hold down ctrl and shift your hand/fingers. Also, If you're holding down ctrl, you're saying you _want_ the tab switcher to stay open while opening the find dialog?

But, I guess, then again, the whole "bad faith actor" is a valid reason to rip this out and just _not_ allow users to use their nT/pT keybinding to navigate.

Reply:
Downloaded the branch and tested it. Yeah, this works fine. And scenarios like "opening the search box while the tab switcher is open" would probably fall under the "play stupid games" category. Fixing this is probably more effort than it's worth right now, so good enough for me!

Reply:
Also, I want this for "dispatch a keybinding while the palette is open". Like, if the user opens the palette, sees the keybinding for "new tab" is ctrl+shift+t, they should be able to just press that and have it do the right thing, you know?